core/src/components/side-menu/side-menu.scss

```diff
@@ -65,14 +66,14 @@
 }
 
 @media (max-width: $grid-lg) {
-  :host([open]) {
+  :host(.menu-opened) {
```

Comment:
Does [open="true"] not work as we want it here?

Reply:
I was expecting it to work the same way as disabled, so to be there but have a value of true or false, but it did not work. Then I tried setting it as hard-coded attribute on <Host, did not work either. Also I have played with removing reflect and putting it back for open prop, did not do much of change. 
Using class in CSS was only thing I could think of as then devs can have "open" prop all the time in the code and just change toggle the value from true to false while CSS will act differently if class is there or not.
Do you have some other idea how we could do it?

Reply:
No other ideas, was more wondering why you went with classes intsead of attr. The only small thing for me would be to change from ".menu-opened" to just ".open"?  And the same for ".menu-persistent" -> ".persistent".

To me it's just to keep the names as short as possible and the prefix of menu- doesn't really add anything since we are already withing the side-menu. What do you think?

Reply:
I will try to shorten it, it was bit off in the beginning as we have props with same names so my IDE was bit confused, but I will double check it. ;)

Reply:
Post-discussion conclusion: we leave it as it for now.